packages/ui/src/ui/pages/query-tracker/Vcs/VcsPath/VcsPathInputForm.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {FC, useCallback, useState} from 'react';
+import {Button, Icon, TextInput} from '@gravity-ui/uikit';
+import CheckIcon from '@gravity-ui/icons/svgs/check.svg';
+import XmarkIcon from '@gravity-ui/icons/svgs/xmark.svg';
+import './VcsPathInputForm.scss';
+import cn from 'bem-cn-lite';
+
+type Props = {
+    path: string;
+    onCancel: () => void;
+    onSave: (path: string) => void;
+};
+
+const block = cn('vcs-path-input-form');
+
+export const VcsPathInputForm: FC<Props> = ({path, onSave, onCancel}) => {
```

Comment:
Usages of the component might be replaced by `components/EditableAsText`

Reply:
This component doesn't fit the logic of this composition